### ISSUE TITLE
Thaumic Augmentation 2.0.9 breaks Cloudstepper ring

### ISSUE THREAD

**Issue by yojakeo** [TheCodex6824/ThaumicAugmentation#117]
Updated Thaumic Augmentation to get the Metaspacial Extruder to work on my server and now the Cloudstepper ring is broken. Both the double jump and the fall reduction seem to be disabled.

**Comment by TechnoMysterio** [comment]
I've had that problem on a server before, and that didn't have Augmentation installed on it. However, it could be Augmentation in this case, although vanilla Thaumcraft is pretty damn buggy so it could also just be another mod or something else entirely. Could you perhaps perform a quick test to see if the ring starts/stops working if you remove/readd Augmentation to the pack (in a test world probably)?

**Comment by DoomRater** [comment]
I am on 2.0.9 and my cloudstepper ring works for double jump.  I can't test fall reduction at the moment though as I'm already wearing a Ring of Odin from Botania which stops ALL fall damage.  I mean I could take it off and test but not tonight.

**Comment by yojakeo** [comment]
I'm assuming it's TA since that's the only mod I updated that deals with Thaumcraft. But I'll make sure to do some more testing to ensure it's actually TA's fault and not one of the other Thaumcraft addons I have.

For the record for addons I have:
Crimson Revelations
Thaumic Augmentation
Thaumic Tinkerer
Thaumic Bases
Thaumic Gadgets
Thaumic Periphery
Thaumic JEI
And inventory scanning for my addons.

**Comment by DoomRater** [comment]
My list of additional mods are as follows:
Augmentation
Thaumic Additions: Reconstructed
Thaumic Periphery
Engineered Golems
Thaumic Wonders
DupeFix Project
Variegated
Rustic Thaumaturgy

Wuss Mode is on and I have CraftTweaker recipes to sidestep that consequence a bit.  I find it interesting our add-ons don't match up much at all.

**Comment by MorzanTLD** [comment]
I only have this issue when the **Jump** key is conflicting with another key such as the impetus thruster keybind. Have you tried unbinding any conflicting key? Additionally, the ring seems to reduce the damage taken from falling but still puts you in hitstun.

**Comment by LannyE** [comment]
Upgrading to 2.0.9 broke my cloudstepper ring on my server.  Downgrading back to 2.0.8 fixed it.  

Addons I have that deal with Thaumcraft are:
Thaumic Augmentation
Crimson Revelations
Thaumic Augmentation
Thaumic JEI
Thaumic Restoration
Thaumic Wands
Thaumic inventory scanning

**Comment by TechnoMysterio** [comment]
Morzan made a good point about the jump key, since 2.0.9 added a configurable keybind for the impetus booster, so if you try remapping that to another key is the issue still there?

**Comment by DoomRater** [comment]
Reproduced immediately by setting the impetus thruster to my jump key.  I did not notice it right away because I remap my jump key to E.

**Comment by TechnoMysterio** [comment]
Ok then, for anyone having the problem you can remap the impetus boost key as a temporary solution to the problem. Not sure when Codex will be able to look into solving the actual issue, but hopefully the temporary solution will work well enough in the meantime.

**Comment by TheCodex6824** [comment]
I thought there was a reason I didn't make it a keybind originally, but unfortunately I missed this when someone recently asked me to add one. Unfortunately there's not really a way to tell the game that it's ok to have the key be conflicting, so I'll probably have to make a normal config option to activate the keybind vs keeping it on jump. So that brings me to the question: which do people prefer more? I'm not sure which should be the default. I personally like it being on jump but I can see how it can be annoying.

So tl;dr I'll add a config option to address this, feedback on what the default should be would be appreciated. In the meantime, ensuring the key is not conflicting with things like jump will fix it.
